tests/TeradataDatatypeTest.php

```diff
@@ -347,4 +347,18 @@ public function invalidLengths(): array
             ['INTERVAL YEAR TO MONTH', '5'],
         ];
     }
+
+    public function testGetTypeByBasetype(): void
+    {
+        $this->assertSame('DATE', Teradata::getTypeByBasetype('DATE'));
```

Comment:
jen poznamka: bylo by imho hezci mit rozelene validni a nevalidni testcase , plus bych v tom expected pouzil nase basetype konstanty. Ale to jsou malickosti